src/main/java/org/zalando/nakadi/repository/TopicRepository.java

```diff
@@ -17,10 +12,14 @@
 import org.zalando.nakadi.exceptions.TopicCreationException;
 import org.zalando.nakadi.exceptions.TopicDeletionException;
 
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
 public interface TopicRepository {
 
-    String createTopic(int partitionCount, Long retentionTimeMs)
-            throws TopicCreationException, DuplicatedEventTypeNameException;
+    String createTopic(int partitionCount, Long retentionTimeMs) throws TopicCreationException;
```

Comment:
Of course this make sense, since the implementation of this interface uses `TopicCreationException` in case of duplicate topics, but I wonder why this is in a PR about timeline lists and deletion?